src/cpp/server/backend_metric_recorder.cc

```diff
@@ -41,16 +42,23 @@ grpc_core::TraceFlag grpc_backend_metric_trace(false, "backend_metric");
 namespace grpc {
 namespace experimental {
 
+std::unique_ptr<ServerMetricRecorder> ServerMetricRecorder::Create() {
+  return std::unique_ptr<ServerMetricRecorder>(new ServerMetricRecorder());
```

Comment:
This can say `return std::make_unique<ServerMetricRecorder>()`.

Reply:
Needed since the ctor is private (the above does not compile). I think in prod we use WrapUnique().